applications/MeshingApplication/custom_processes/mmg_process.cpp

```diff
@@ -1227,7 +1228,7 @@ void MmgProcess<TMMGLibrary>::CleanSuperfluousNodes()
     const auto it_elem_begin = r_elements_array.begin();
 
     // Saving the nodes that belong to an element
-    #pragma omp parallel for
+    // #pragma omp parallel for
```

Comment:
I think you pushed this change by error. You can `git checkout` the `master` version of the file.